### PR TITLE
remove futuretense/lock-manager

### PR DESCRIPTION
**Pull request by FutureTense**

### CODE REVIEW

**Comment by FutureTense** [comment]
@ludeeus I’m not sure what to do about this “initialize” error.  Can you advise?

**Comment by KTibow** [comment]
The checks are built for new repos, not removing repos.

**Comment by FutureTense** [comment]
@KTibow so there is nothing left for me to do here?

**Comment by KTibow** [comment]
Except wait for ludeeus to come around.

File: removed [hunk 1]
```diff
@@ -405,5 +405,11 @@
     "reason": "NWS radar server no longer operational",
     "removal_type": "removal",
     "link": "https://github.com/MatthewFlamm/nwsradar/issues/16"
+  },
+  {
+    "repository": "futuretense/lock-manager",
+    "reason": "replacing integration",
+    "removal_type": "integration",
```
Comment (ludeeus) [inline removed:412]:
```suggestion
    "removal_type": "removal",
```

**Comment by FutureTense** [comment]
@ludeeus is this ok?

**Review by raman325 — APPROVED** [review]

**Pull request merged**